packages/ui-core/theme/src/lib/components/user/user.component.html

```
@@ -1,12 +1,28 @@
+		[class.with-identity]="showIdentity"
+		[attr.title]="showIdentity ? null : user?.name"
+		[attr.aria-label]="user?.name"
+		role="button"
+		tabindex="0"
+		(click)="onClicked()"
+		(keydown.enter)="onClicked()"
+		(keydown.space)="onClicked()"
+	>
+		<div class="img-container">
+			@if (user?.imageUrl) {
+				<img [src]="user.imageUrl" [alt]="user?.name" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)